File: backend/src/main/java/com/expensesplitter/settlement/dto/SettlementResponse.java
package com.expensesplitter.settlement.dto;

import lombok.*;

import java.math.BigDecimal;
import java.time.LocalDateTime;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class SettlementResponse {

    private Long id;
    private Long paidById;
    private String paidByName;
    private Long paidToId;
    private String paidToName;
    private BigDecimal amount;
    private LocalDateTime createdAt;
}

File: android/app/src/main/java/com/expensesplitter/app/model/settlement/SettlementResponse.kt
package com.expensesplitter.app.model.settlement

import java.math.BigDecimal

data class SettlementResponse(
    val id: Long,
    val paidById: Long,
    val paidByName: String,
    val paidToId: Long,
    val paidToName: String,
    val amount: BigDecimal,
    val createdAt: String
)
